lib/internal/blob.js

```diff
@@ -311,7 +312,10 @@ class Blob {
     if (!isBlob(this))
       throw new ERR_INVALID_THIS('Blob');
 
-    const dec = new TextDecoder();
+    if (dec === undefined) {
+      dec = new TextDecoder();
+    }
```

Comment:
```suggestion
    dec ??= new TextDecoder();
```

Reply:
Well, your solution is actually faster. Thanks for the review @aduh95. I'll update as soon as possible.

Here's the benchmark: https://jsben.ch/wtmnh

Reply:
Updated the pull request.